src/TextField/TextField.js

```diff
@@ -132,7 +132,7 @@ function TextField(props) {
 TextField.propTypes = {
   /**
    * This property helps users to fill forms faster, especially on mobile devices.
-   * The name can be confusion, it's more like an autofill.
+   * The name can be confusing, it's more like an autofill.
    * You can learn about it with that article
    * https://developers.google.com/web/updates/2015/06/checkout-faster-with-autofill
    */
```

Comment:
Thank for your first PR! 👍 

While you're correcting this, lets make a couple of other changes:

```diff
+   The `autocomplete` attribute of the `input` element.
    * This property helps users to fill forms faster, especially on mobile devices.
-   * The name can be confusion, it's more like an autofill.
+   * The name can be confusing, as it's more like an autofill.
-   * You can learn about it with that article
-   * https://developers.google.com/web/updates/2015/06/checkout-faster-with-autofill
+   * You can learn more about it here:
+   * https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofill
```

Please could you also fix the same prop description in the Input component to match? Just in case you were editing it manually, you can update the `md` files by running:

```
npm run docs:api
```

Reply:
Interestingly the Input docs already said "confusing", however there were other discrepancies between the two.  The docs from Input had some better grammar regarding the linked google article, so I updated the PR with a verbatim copy of the `Input.autoComplete` docs into `TextField.autoComplete`

Reply:
Yup, I noticed that too. I suspect one has been improved at some point, without knowledge of the errors elsewhere.

If you wouldn't mind following the wording I proposed for both though, as the whatwg page is part of the html spec, rather than an article; and we're still missing the other corrections to the previous sentences.

Thanks! 👍

Reply:
Sorry I didn't fully notice you had proposed changes to the content other then making them the same.  I've updated the PR again, hopefully with what you expected this time 👍